docs/TESTS.md

```diff
@@ -1,112 +1,65 @@
-## Xcode Tutorial
+## Testing Tutorial
 
-This guide explains how to create and configure a Swift-based Xcode project for Exercism.
+This guide explains how to run unit tests for Exercism Swift exercises, and optionally how to use Xcode for development.
 
-Apple does not support user-defined project templates in Xcode, and the default templates are more elaborate than those needed for Exercism. This guide bridges that gap.
+Exercism uses [Swift Package Manager](https://github.com/apple/swift-package-manager/tree/master/Documentation) to package files and tests for Swift.
```

Comment:
Let's make whatever changes we can to make this more approachable for both beginners and Linux users.  I think we clearly need to document both the Xcode and CLI use cases, but we can probably do more to explain why you would use one over the other.  @bobdel could you comment on specific lines of the changeset that are confusing?